README.md

```diff
@@ -151,6 +151,19 @@ class Config:
 ```
 When indicated, this field is not added to the parser and cannot be overridden with an argument.
 
+### init=False
+Fields marked with init=False will be ignored by pydargs and cannot be overridden with an argument.
+
+```python
+@dataclass
+class Config:
+    number: int
+    ignored: str = field(init=False)
+```
+
+Additionally, this field will not be present on the config object at all. It is recommended to add a [__post_init__ method](https://docs.python.org/3/library/dataclasses.html#post-init-processing) to the dataclass to set the value of the field.
```

Comment:
```suggestion
This could be useful in combination with a [`__post_init__()` method](https://docs.python.org/3/library/dataclasses.html#post-init-processing) to set the value of the field.
```

I don't think it is our place to do recommendations on how to use dataclasses, but adding the link is valuable.